jdbc/src/main/java/io/kestra/jdbc/repository/AbstractJdbcLogRepository.java

```diff
@@ -98,7 +98,7 @@ private <T extends Record> SelectConditionStep<T> filter(
         }
 
         if (startDate != null) {
-            select = select.and(field("timestamp").greaterOrEqual(startDate.toOffsetDateTime()));
```

Comment:
I would not change that here, I suspect it's because of the log shipper right?
I would increase the offset date to 1ms (or 1 nano) in the log shipper instead.